lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -261,6 +263,7 @@ struct FuncOpConversion : public FuncOpConversionBase {
     // for `nvvm.annotation` metadata.
     newFuncOp->setAttr(NVVMMetadataField::MaxNTid,
                        rewriter.getIntegerAttr(i32_ty, 32 * NumWarps));
+#endif
 
```

Comment:
Is there any equivalent for Rocm? This is useful for multi kernels. 
If to be done, let's put a TBD for future installment.